### DISCUSSION TITLE
can someone compile mpv for me with a libplacebo patch? (HDR-tonemapping fix)

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15358] [Q&A] [answered]
Hi, I need mpv compiled with a new [fix in libplacebo](https://code.videolan.org/videolan/libplacebo/-/merge_requests/683/diffs?diff_id=853375) that fixes HDR-tonemapping. Can someone help me with that? I never compiled anything and this is a bit too complicated for me. It can take weeks/months until this fix gets merged and pushed etc.

I'm on Windows 10 and use the x86_64-v3 build

Thank you :)

**Comment by kasper93** [comment, marked as answer]
You will find a build here https://github.com/kasper93/mpv/actions/runs/11980106573 once it finishes.

**Reply by geextahslex** [reply]
Thank you very much :) Do you know why the .exe is 42mb when the normal mpv has 107mb?

**Reply by kasper93** [reply]
I don't know what the "normal mpv" is, but likely there are more 3rdparty libraries statically linked which makes it bigger.
